kornia/augmentation/_2d/intensity/random_rain.py

```diff
@@ -36,12 +36,12 @@ class RandomRain(IntensityAugmentationBase2D):
 
     def __init__(
         self,
-        same_on_batch: bool = False,
-        p: float = 0.5,
-        keepdim: bool = False,
         number_of_drops: tuple[int, int] = (1000, 2000),
         drop_height: tuple[int, int] = (5, 20),
         drop_width: tuple[int, int] = (-5, 5),
+        same_on_batch: bool = False,
```

Comment:
i don't know why this is in the "wrong" sequence, it's a breaking change, but we should do it

Reply:
Sure